.github/workflows/full-tests-linux.yml

```
@@ -22,15 +22,22 @@ jobs:
         pip install pytest
         pip install pytest-cov
     - name: Get ephy_testing_data current head hash
-      id: vars
+      id: ephys
       run: echo "::set-output name=HASH_EPHY_DATASET::$(git ls-remote https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git HEAD | cut -f1)"
-    - uses: actions/cache@v2
+    - name: Cache ephys dataset - ${{ steps.ephys.outputs.HASH_EPHYS_DATASET }}
+      uses: actions/cache@v2
       id: cache-datasets
       with:
         path: /home/runner/work/nwb-conversion-tools/nwb-conversion-tools/ephy_testing_data
-        key: ecephys-dset-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
-        restore-keys: ecephys-dset-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
-    - name: Run pytest with coverage
+        key: ecephys-dset-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
+        restore-keys: ecephys-dset-${{ steps.ephys.outputs.HASH_EPHY_DATASET }}
+    - name: Set environment variables
+      run: |
+        echo "TEST_ENV=ci" >> $GITHUB_ENV
```

```suggestion
    - name: Set environment variables
      run: echo "TEST_ENV=ci" >> $GITHUB_ENV
```
No need to break over multiple lines unless there are multiple commands